.github/workflows/ci-for-diff-ns.yaml

```diff
@@ -44,6 +43,15 @@ jobs:
           kubectl rook_ceph -o test-operator -n test-cluster rook status
           kubectl rook_ceph -o test-operator -n test-cluster rook status all
           kubectl rook_ceph -o test-operator -n test-cluster rook status cephobjectstores
+          # to allow time for reconcile, sleep before listing the pools
+          sleep 5
+          kubectl rook_ceph -o test-operator -n test-cluster rbd ls replicapool
+
+          # for testing osd purge scale the osd deplyment
+          kubectl --namespace test-cluster scale deploy/rook-ceph-osd-0 --replicas=0
+          # we need to sleep so the osd will be marked down before purging the osd
+          sleep 5
+          kubectl-rook_ceph -o test-operator -n test-cluster rook purge-osd 0 --force
```

Comment:
one of the ci will test `--force` and the other will be running without `--force` so that we test both scenarios.

Reply:
passing `--force` in both the ci test since the osd is not out, it will not able to purge the osd, and the operator will keep waiting for osd to be out and CI will keep on running forever

Reply:
yeah, with the small cluster we will need to force